### PR TITLE
Align information to restructured project

### PR DESCRIPTION
**Pull request by egekorkan**
Basically, it aligns our page to the restructuring. With more time, we should do it definitely better. I wanted to get something out there that can be reviewed and merged before EclipseCon. #52 should be merged first

Note: Please review with white spaces hidden. There was a mixed usage of tabs and spaces and I have made them all spaces in files I have touched.

fixes https://github.com/eclipse-thingweb/website/issues/48

### CODE REVIEW

File: themes/thingweb/layouts/index.html [hunk 1]
```diff
@@ -2,37 +2,99 @@
...
+                            This gives developers the flexibility to choose the specific tools they need for their IoT
+                            solution without sacrificing interoperability.
+                        </p>
+
+                        <div>
+                            <h2>node-wot</h2>
+
+                            <p>
+                                The Eclipse Thingweb™ <code>node-wot</code> is the official reference implementation of
+                                the Web of Things (WoT) Scripting API written in the <a
```
Comment (danielpeintner) [inline themes/thingweb/layouts/index.html:52]: for the official specification?

**Review by danielpeintner — COMMENTED** [review]
Some comments:
* The text "Open-source Web of Things Projects" is not right for me. Why do we say "Open-Source". Don't we want to say something like "Thingweb Web of Things Projects"
* The aforementioned text should remain a "header" .. otherwise the gear icon looks lost and the text  is to long also (try to resize the window)

I think the rest looks good to me 👍

File: themes/thingweb/layouts/index.html (hunk 1 above)
Comment (egekorkan) [reply themes/thingweb/layouts/index.html:52]: Do you mean removing "official"? We can also just remove the whole thing starting with "written"?

**Comment by egekorkan** [comment]
> * The text "Open-source Web of Things Projects" is not right for me. Why do we say "Open-Source". Don't we want to say something like "Thingweb Web of Things Projects"

I thought way too long about this and I am still not satisfied :D So the original problem was that the text was too long and overflowing to the right and thus making it difficult to read. I did not want to repeat "Thingweb" since it is written on the top left anyways. I wrote open-source since someone can come to this website without knowing that it is an Eclipse Project. I am open for suggestions but an adjective would be good. Some other ideas I had were:

1. Interoperable Web of Things Projects
2. Open-source Web of Things Implementations (Implementations is too long though)
3. Web of Things at Eclipse
4. Web of Things tools for all

> * The aforementioned text should remain a "header" .. otherwise the gear icon looks lost and the text  is to long also (try to resize the window)

I did not completely understand this one. Do you mean we should repeat the "Open-Source Web of Things Projects"? I would be also fine to remove the text from the image and only keep it in the header here. Or we can put Eclipse Thingweb in the header.

**Comment by egekorkan** [comment]
Above the gear icon: Web of Things Components

The catchphrase: 
- Innovative and Interoperable Components for **your** IoT solution (too long)

**Pull request merged**